lifelines/statistics.py

```diff
@@ -208,7 +208,7 @@ def chisq_test(U, degrees_freedom, alpha):
 
 
 def two_sided_z_test(Z, alpha):
-    p_value = 1 - max(stats.norm.cdf(Z), 1 - stats.norm.cdf(Z))
+    p_value = 1 - np.max(stats.norm.cdf(Z), 1 - stats.norm.cdf(Z))
```

Comment:
Small arrays are irrelevant as the overhead to go into C with numpy will win out every time.
For large data however:
![image](https://cloud.githubusercontent.com/assets/223655/6042032/f30793e4-ac81-11e4-8a61-27d63e171b8c.png)